pkg/virt-controller/services/template.go

```diff
@@ -1608,3 +1593,22 @@ func copyProbe(probe *v1.Probe) *k8sv1.Probe {
 		},
 	}
 }
+
+func alignPodMultiCategorySecurity(pod *k8sv1.Pod, selinuxType string) {
+	pod.Spec.SecurityContext.SELinuxOptions = &k8sv1.SELinuxOptions{Type: selinuxType}
+	for i := range pod.Spec.Containers {
+		container := &pod.Spec.Containers[i]
+		if container.Name != "compute" {
```

Comment:
I know that you do not introduce it here, but why do we skip the "compute" container?  Can you make the code the answer more apparent from the code (possibly in a follow-up pr)?

Besides, it feels like the string `"compute"` is used too much in code and tests. Can you turn it into a constant (or function) with a meaningful name (not in this PR)?

Reply:
+1
As the comment has been changed, I think explaining this here makes sense as well.

Reply:
It's an selinux internal: processes labeled having MCS can access contexts without MCS, but not the other way around.

To allow the compute container to access stuff in the other containers in the same pod (since their MCS labels are different when running on moby), we remove the MCS part from the selinux context.

I can try to make this more obvious to the reader in a follow-up PR, but I'm unsure such a deep detail deserves to be made apparent in the code.